File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        int a = num_list[0];
        int b = 0;
        
        for(int i=1;i<num_list.length;i++){
            if(i % 2 == 0){
                a += num_list[i];
            }else{
                b += num_list[i];
            }
        }
        answer = a > b ? a : b;
        return answer;
    }
}

File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.py
def solution(num_list):
    x = 0;
    y = 0;
    
    for i in range(0, len(num_list)):
        if i % 2 == 0:
            x += num_list[i];
        else:
            y += num_list[i];
            
    answer = x >= y and x or y;
    
    return answer